### ISSUE TITLE
No resources found for profileboundle and profile

### ISSUE THREAD

**Issue by hugohdz89** [openshift/compliance-operator#455]
Hello

I just installed the operator in a OCP 4.X

However, when trying to get the output of bellow commands, is says there are no resources.

$ oc get profilebundle.compliance
No resources found in openshift-compliance namespace.

$ oc get profile.compliance
No resources found in openshift-compliance namespace.

What can be the issue?

**Comment by hugohdz89** [comment]
I re-deployed all in the namespace, now I get the resources, however in the console, I see this and not sure if this is normal/expected.

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/16210206/93918089-7a649d00-fcd1-11ea-9898-c1702eddf331.png">

**Comment by JAORMX** [comment]
@hugohdz89 most likely it's just that the operator is still parsing the profiles. They don't come up immediately, unfortunately.

Related to the upgradePending issue, I'm not sure.

**Comment by hugohdz89** [comment]
Hi @JAORMX thanks for the response!
The profiles appeared after some minutes.

However I sill see the UpgradePending, 
![image](https://user-images.githubusercontent.com/16210206/94025169-d553ce00-fd7d-11ea-8b36-9e483394174e.png)

Under Installed Operators, I see it says 0.1.17 version, however in the Operator Hub it says a different version

![image](https://user-images.githubusercontent.com/16210206/94025298-f87e7d80-fd7d-11ea-8059-83359d23c751.png)


I've used the latest version of this repo, which from my understanding must being using the 0.1.17 version.

**Comment by JAORMX** [comment]
that's strange... I tried the installation right now and wasn't able to reproduce it.

**Comment by hugohdz89** [comment]
Hello @JAORMX 

We've already updated our cluster to 4.5.13 but still see the same issue with Operator, however the operator has been updated to 0.0.18 version.
![image](https://user-images.githubusercontent.com/16210206/96295320-4e2bfb80-0fb3-11eb-9a04-f2ee82c9e96d.png)

Another issue is that when the remediation is applied, it changes the status to "Applied" and the machine-config object is created, however the machineconfigpool status changes to Degraded with this message.
**Failed to render configuration for pool worker: parsing Ignition config failed with error: unsupported config version Report:**

[hhmoran@ocpjump ~]$ oc get mcp -w
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-6c9cb2bad143834f4fde9cdb9e1b78fb   True      False      False      3              3                   3                     0                      39d
worker   rendered-worker-a6cf7a2c0df1941ba4dbc7165c3a9799   True      False      True       3              3                   3                     0                      39d

I appreciate your response!
